docs/reference/meson-compatibility.rst

```diff
@@ -39,6 +39,14 @@ versions.
    Meson 1.3.0 or later is required for compiling extension modules
    targeting the Python limited API.
 
+.. option:: 1.6.0
+
+   Meson 1.6.0 or later is required to support ``license`` and
```

Comment:
Do you have a link to a PR or something for the 1.6 change? curious as to what needed updates in meson, was expecting it to be just meson-python.

Reply:
Probably due to depending on https://github.com/mesonbuild/meson/pull/13783

Reply:
Indeed, as @eli-schwartz says. Meson support is required only to get the license and license files declared in `meson.build` when they are declared dynamic or when there is no `project` section in `pyproject.toml`.